index.js

```diff
@@ -68,7 +68,52 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+
+header.addEventListener('click', function() {
+  toggleColor(header)
+})
+
+// header.addEventListener('click', toggleColor)
+// why doesn't this work?
```

Comment:
The `toggleColor` function needs to be called with an **element**, but when you pass it as a callback to the event listener, it will be called with the **event**. This would work:

```js
function toggleColor(event) {
  if (event.target.style.color === "red") {
    event.target.style.color = "black"
  } else {
    event.target.style.color = "red"
  }
}
const header = document.querySelector("#header")
header.addEventListener("click", toggleColor)